Makefile

```diff
@@ -13,18 +13,19 @@ install: os_migrate-os_migrate-latest.tar.gz
 	if [ -n "${VIRTUAL_ENV:-}" ]; then \
 		source /root/venv/bin/activate; \
 	fi; \
+        cd releases/
```

Comment:
This will not work, there would have to be ; \ at the end of the line (see the line above). Nit: i'd just alter the path on the line below with releases/ instead of doing `cd` :). So this one needs a tweak, otherwise all LGTM.

Reply:
Done